__tests__/components/measure-upload/DateSelectors.test.tsx

```diff
@@ -15,52 +12,11 @@ describe('DateSelectors', () => {
     }));
   });
 
-  it('renders two date selectors with proper labels', () => {
+  it('renders a date range picker with proper labels', () => {
     render(mantineRecoilWrap(<DateSelectors />));
 
-    const periodStartSelector = screen.getByLabelText('Measurement Period Start');
-    expect(periodStartSelector).toBeInTheDocument();
-    const periodEndSelector = screen.getByLabelText('Measurement Period End');
-    expect(periodEndSelector).toBeInTheDocument();
-  });
-  it('Updates measurementPeriodState when period start date selected', async () => {
-    const MockMP = getMockRecoilState(measurementPeriodState, {
-      start: DateTime.fromISO('2020-01-01').toJSDate(),
-      end: DateTime.fromISO('2021-01-01').toJSDate()
-    });
-    render(
-      mantineRecoilWrap(
-        <>
-          <MockMP />
-          <DateSelectors />
-        </>
-      )
-    );
-    const periodStartSelector = screen.getByPlaceholderText('Select start date');
-    await act(async () => {
-      fireEvent.change(periodStartSelector, { target: { value: DateTime.fromISO('2019-01-01').toJSDate() } });
-    });
-    const dateDisplay = screen.getByDisplayValue('January 1, 2019');
-    expect(dateDisplay).toBeInTheDocument();
-  });
-  it('Updates measurementPeriodState when period end date selected', async () => {
-    const MockMP = getMockRecoilState(measurementPeriodState, {
-      start: DateTime.fromISO('2020-01-01').toJSDate(),
-      end: DateTime.fromISO('2021-01-01').toJSDate()
-    });
-    render(
-      mantineRecoilWrap(
-        <>
-          <MockMP />
-          <DateSelectors />
-        </>
-      )
-    );
-    const periodEndSelector = screen.getByPlaceholderText('Select end date');
-    await act(async () => {
-      fireEvent.change(periodEndSelector, { target: { value: DateTime.fromISO('2022-01-01').toJSDate() } });
-    });
-    const dateDisplay = screen.getByDisplayValue('January 1, 2022');
-    expect(dateDisplay).toBeInTheDocument();
+    const periodSelector = screen.getByLabelText('Measurement Period Range');
+    expect(periodSelector).toBeInTheDocument();
   });
+  //TODO: Add unit testing for updates on change
```

Comment:
I have been looking at testing for this for a long time now and not have that much luck...one thing you could add but I am not sure how helpful it is is just one test (like below) that tests if the dates are displayed when there is a measurement period. I will keep investigating for some others. 
![Screen Shot 2022-10-03 at 2 24 13 PM](https://user-images.githubusercontent.com/30158156/193651025-e86181b1-b57c-40d7-a432-638159b54613.png)